### ISSUE TITLE
Control GPU Usage Percentage

### ISSUE THREAD

**Issue by BaoNguyen3001** [AlexeyAB/darknet#1159]
Hi everyone,
I am trying to find existing thread about "How to control GPU usage percentage", but did not find the answers.
So could you please help me in "control percentage of GPU usage" in:
- Python wrapper provided in this repo
- In C-API of darknet
I only want to use ~50% of GPU for Yolo.

Thank you so much!

**Comment by aimhabo** [comment]
GPU usage is bound to `width*height*channels*batch/subdivisions` in .cfg. It is best to adjust according to this relationship

**Comment by BaoNguyen3001** [comment]
Hi, thank you for you reply!
However, this is not really what I expected.
I want to ask if darknet supports any functions such like (in tensorflow):
- config.gpu_options.allow_growth = True  ===> to allow grow memory to fit remaining memory
- config.gpu_options.per_process_gpu_memory_fraction = 0.7 ===> to control the amount of memory

Thank you!

**Comment by AlexeyAB** [comment]
@BaoNguyen3001 No, you can't contol amount of Gigabytes of GPU-RAM-usage directly. You can only increase `subdivisions=16` 32 or 64 to decrease GPU-RAM-usage.
